includes/wsuwp-shortcode-fundselector-utils.js

```diff
@@ -265,9 +265,15 @@ var EPSILON = 0;
 				};
 				
 				jQuery("#advFeeAmount").text(_.template(wpData.adv_fee_message)({advFeeAmount: advFeeDecimal.toFixed(2) }));
+				jQuery("#advFeeAmount").html(jQuery("#advFeeAmount").html().replace("Advancement Fee", '<a href="https://foundation.wsu.edu/about/fee-disclosure/" target="_blank">Advancement Fee</a>'));
+				jQuery("#advFeeAmount").html(jQuery("#advFeeAmount").html() + '<a id="moreAdvFeeInfo" href="https://foundation.wsu.edu/about/fee-disclosure/" target="_blank"></a>');	
```

Comment:
The updated link is now https://foundation.wsu.edu/fees/, based on feedback from Trevor.